dali/python/nvidia/dali/types.py

```diff
@@ -54,14 +54,16 @@ def _not_implemented(val):
         DALIDataType.DATA_TYPE : ("nvidia.dali.types.DALIDataType", lambda x: DALIDataType(int(x))),
         DALIDataType.INTERP_TYPE : ("nvidia.dali.types.DALIInterpType", lambda x: DALIInterpType(int(x))),
         DALIDataType.TENSOR_LAYOUT : (":ref:`layout str<layout_str_doc>`", lambda x: str(x)),
-        DALIDataType.PYTHON_OBJECT : ("object", lambda x: x)
+        DALIDataType.PYTHON_OBJECT : ("object", lambda x: x),
+        DALIDataType._TENSOR_LAYOUT_VEC : (":ref:`layout str<layout_str_doc>`", _to_list(lambda x: str(x)))
```

Comment:
This isn't needed right?
It looks like it works fine through _vector_types.

Reply:
Vector types still need to be listed here. The entry points to a convert function that is used, e.g. in 
```python
def _type_convert_value(dtype, val):
    if dtype not in _known_types:
        raise RuntimeError(str(dtype) + " does not correspond to a known type.")
    return _known_types[dtype][1](val)
```

Reply:
yup, it's needed.